### PR TITLE
Add Pradyumna Gayake to the list

### PR DESCRIPTION
**Pull request by prgayake**

### CODE REVIEW

File: public/data/prgayake.json [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+      "url": "https://github.com/prgayake",
+      "icon": "github"
+    },
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://mobile.twitter.com/pradyumna__g",
+      "icon": "twitter"
+    }
+
+
```
Comment (Panquesito7) [inline public/data/prgayake.json:17]:
```suggestion
```

File: public/data/prgayake.json [hunk 2]
```diff
@@ -0,0 +1,30 @@
+{
+  "name": "Pradyumna Gayake",
+  "bio": "Web Developer 🖥 | Passionate about coding ",
```
Comment (Panquesito7) [inline public/data/prgayake.json:3]:
```suggestion
  "bio": "Web Developer 💻 | Passionate about coding",
```

File: public/data/prgayake.json [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+  "milestones": [
+    {
+      "title": "Runner up at Microsoft Azure Developer League 2021",
+      "date": "Aug 2021",
+      "icon": "microsoft",
+      "color": "blue",
+      "description": "Runner up at Microsoft Azure Developer League 🏆🎖",
+      "url": "https://www.hackerearth.com/challenges/hackathon/azure-developer-league-hackathon/"
+    }
+
```
Comment (Panquesito7) [inline public/data/prgayake.json:28]:
```suggestion
```

**Review by Panquesito7 — CHANGES_REQUESTED** [review]

**Comment by prgayake** [comment]
Hey There, 
Done all the Requested changes and Waiting for your response.

**Review by rohan-kulkarni-25 — APPROVED** [review]

**Pull request merged**